src/nimony/reporters.nim

```diff
@@ -108,6 +110,11 @@ proc reportErrors*(dest: var TokenBuf): int =
     if dest[i].kind == ParLe and dest[i].tagId == errTag:
       inc result
       let info = dest[i].info
+      if reportedErrSources.containsOrIncl(info):
+        let x = cursorAt(dest, i)
+        inc i, span(x)
+        endRead(dest)
+        continue
```

Comment:
No. Instead just prevent the `r.error` and `r.trace` calls from happening.

Reply:
done